app/cdap/components/common/BoxContainer/index.tsx

```diff
@@ -25,161 +25,101 @@ const SimpleBox = styled(Box)`
   width: ${({ width }) => (width ? width : 'auto')};
   height: ${({ height }) => (height ? height : 'auto')};
   box-sizing: border-box;
-  background-color: ${({ backgroundColor }) => (backgroundColor ? backgroundColor : 'transparent')};
   padding: ${({ padding }) => (padding ? padding : 0)};
   margin: ${({ margin }) => (margin ? margin : 0)};
-  border-radius: ${({ borderRadius }) => (borderRadius ? borderRadius : 0)};
-  border: ${({ border }) => (border ? border : 0)};
-  text-align: ${({ textAlign }) => (textAlign ? textAlign : 'left')};
 `;
 
 export const BlockContainer = ({
   children,
   padding,
   margin,
-  backgroundColor,
-  borderRadius,
-  border,
   width,
   height,
   onClick,
   dataTestId,
-  textAlign,
 }: IBoxContainerProps) => (
   <SimpleBox
     padding={padding}
     margin={margin}
-    backgroundColor={backgroundColor}
-    borderRadius={borderRadius}
-    border={border}
     width={width}
     height={height}
     onClick={onClick}
     dataTestId={dataTestId}
-    textAlign={textAlign}
   >
     {children}
   </SimpleBox>
 );
 
-const AbsoluteDiv = styled(SimpleBox)`
-  position: absolute;
-  top: ${({ top }) => (top ? top : 0)};
-  left: ${({ left }) => (left ? left : 0)};
-  right: ${({ right }) => (right ? right : 0)};
-  bottom: ${({ bottom }) => (bottom ? bottom : 0)};
+const CenterAligned = styled(SimpleBox)`
+text-align: center;
 `;
 
-export const AbsoluteContainer = ({
+
+export const CenterAlignedBox = ({
   children,
   padding,
   margin,
-  backgroundColor,
-  right,
-  left,
-  top,
-  bottom,
-  borderRadius,
-  border,
   width,
   height,
-  onClick,
   dataTestId,
-  textAlign,
 }: IBoxContainerProps) => (
-  <AbsoluteDiv
+  <CenterAligned
     padding={padding}
     margin={margin}
```

Comment:
why do we have the styling props again
can we just return some styled-components with fixed styles plz